src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate: function(callback, value) {
+      this.result = callback(value, this.result);
+
+      return this;
+    },
+
+    add: function(value, result) {
+      return result + value;
+    },
+
+    subtract: function(value, result) {
+      return result - value;
+    },
```

Comment:
```suggestion
    subtract: function(value) {
      return this.result - value;
    },
```
Try to make it work like that